src/main/java/uk/gov/hmcts/cmc/claimstore/config/DbConfiguration.java

```diff
@@ -48,6 +49,11 @@ public ClaimRepository claimRepository(DBI dbi) {
         return dbi.onDemand(ClaimRepository.class);
     }
 
+    @Bean
+    public ClaimSearchRepository claimSearchRepository(DBI dbi) {
```

Comment:
I would name it `CaseRepository`. That interface would then have two implementations:
- DB based one
- CCD based one